website/content/maintainers/github/templates/required/governance-maintainer.md

```diff
@@ -1,11 +1,216 @@
+are expected to follow these values.  Deciding your values is a good topic
+for a general community meeting.
+
+See our documentation on [Charters] for some examples.
+
+### Maintainers
+
+This section outlines the definition and responsibilities of a Maintainer, 
+which in this governance is identical to being a merge approver (also known
+as a "committer"). Since your Maintainers are are the only authority in the project,
+it is critical to document how that authority is earned.
+
+Your Maintainers should customize the list of requirements to become a Maintainer with requirements that match their actual qualifications. It's better if you can retain
+a path for substantial non-code contributors, such as documentation leads and 
+community managers, to also become Maintainers. 
+
+If you have a separate, full [Contributor Ladder], you will want to cut this 
+list of requirements and refer to that document instead.
+
+This section covers both how new Maintainers are selected, and how existing Maintainers
+may be removed.  Removal is critically important to prevent project paralysis
+when Maintainers move on to other things.
+
+### Meetings
+
+You need to explicitly describe the venues at which official project decisions
+get made. This may happen at a public weekly developer meeting, through GitHub 
+issues and PRs, on a mailing list, or all of those.  This is both so that contributors
+know where to go with requests, and so that they are reassured that important
+project decisions will not happen in the private meeting rooms of any Maintainers' 
+employer.
+
+As such, you may need to modify this section to cover your actual communications
+channels.
+
+### CNCF Resources
+
+This covers one of the powers of Maintainers, which is that they can ask the
+CNCF to do things for the project.  This outlines one possible process for this 
+to happen, based on community meetings.  If you prefer to handle this by GitHub
+instead, the language would be something like this:
+
+```
+Any Maintainer may suggest a request for CNCF resources, by filing an issue in
+the [community repo](TODO:main or /community repository URL).  A simple majority 
+of Maintainers approves the request.  The Maintainers
+may also choose to delegate working with the CNCF to non-Maintainer community
+members, who will then be added to the [CNCF's Maintainer List](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
+for that purpose.
+```
+
+### Code of Conduct
+
+This text assumes that your Maintainer Council themselves is the first recipient
+of most CoC reports.  If you wish reports to go straight to the CNCF Code of Conduct
+Committee without the involvement of maintainers, then use this language
+instead:
+
+```
+[Code of Conduct](./code-of-conduct.md)
+violations by community members will be referred to the CNCF Code of Conduct 
+Committee. Should the CNCF CoCC need to work with the project on resolution, the
```

Comment:
```suggestion
Committee. Should the CNCF CoC Committee need to work with the project on resolution, the
```

I recommend sticking with the most well recognized acronym, CoC, and not having two similiar looking ones for public facing docs.